instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import PostsContainer from './PostsContainer';
+import SearchBar from '../SearchBar/SearchBarContainer';
+
+class PostsPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
+      filteredPosts: []
+    };
+  }
+  componentDidMount() {
+    this.setState({ posts: dummyData });
+  }
+
+  searchPostsHandler = event => {
+    const filteredPosts = this.state.posts.filter(posts =>
+    posts.username === event.target.value);
+  
+    this.setState({ filteredPosts: posts });
```

Comment:
If you're declaring `filteredPosts` on line 19, shouldn't you make use of it? Isn't that what you mean to set in the state like this: `this.setState({ filteredPosts: filteredPosts });`? (In which case, you can use the shorthand: `this.setState({ filteredPosts });`.)